src/sentry/conf/server.py

```diff
@@ -1407,8 +1413,6 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "organizations:incidents": False,
     # Enable issue platform
     "organizations:issue-platform": False,
-    # Whether to allow issue only search on the issue list
-    "organizations:issue-search-allow-postgres-only-search": False,
```

Comment:
do you need to rebase your branch? not sure why these changes are here

Reply:
looks like a bad merge, i'll fix this